### PR TITLE
Array.map returns an array, no need to create one and push

### PR DESCRIPTION
**Pull request by bmclaughlin**
Refactoring out a bad pattern.

### CODE REVIEW

File: src/SmartComponents/Rules/ListRules.js [hunk 1]
```diff
@@ -31,24 +31,21 @@ class ListRules extends React.Component {
...
-                        summary= { value.summary_html }
-                        impact = { value.impact.impact }
-                        likelihood = { value.likelihood }
-                        totalRisk = { value.severity }
-                        riskOfChange = { value.resolution_risk }
-                        ansible = { value.ansible }
-                        hitCount = { value.impacted_systems }
-                    />
-                );
+            const cards = rules.map((value, key) => {
```
Comment (AllenBW) [inline src/SmartComponents/Rules/ListRules.js]: we can take this a step further.... 
```suggestion
            const cards = rules.map((value, key) => <RulesCard
                key = { key }
                widget-id= { value }
                ruleID = { value.rule_id }
                category= { value.category.name }
                description= { value.description }
                summary= { value.summary_html }
                impact = { value.impact.impact }
                likelihood = { value.likelihood }
                totalRisk = { value.severity }
                riskOfChange = { value.resolution_risk }
                ansible = { value.ansible }
                hitCount = { value.impacted_systems }
            />);
```

File: src/SmartComponents/Actions/ActionsOverview.js [hunk 2]
```diff
@@ -48,31 +48,27 @@ class ActionsOverview extends Component {
...
-                            { children }
-                        </Link>
-                    }>
-                    <SummaryChartItem
-                        name={ sevNames[key] }
-                        numIssues={ value }
-                        totalIssues={ this.state.total }/>
-                </ConditionalLink>
-            );
+        let SummaryChartItems = this.state.severity.map((value, key) => {
```
Comment (AllenBW) [inline src/SmartComponents/Actions/ActionsOverview.js]:
```suggestion
        let SummaryChartItems = this.state.severity.map((value, key) =>
            <ConditionalLink
                key={ key }
                condition={ value }
                wrap={ children =>
                    <Link to={ `/actions/${sevNames[key].toLowerCase()}-risk` }>
                        { children }
                    </Link>
                }>
                <SummaryChartItem
                    name={ sevNames[key] }
                    numIssues={ value }
                    totalIssues={ this.state.total }/>
            </ConditionalLink>
        );
```

also this one....

File: src/SmartComponents/Actions/ActionsOverview.js [hunk 3]
```diff
@@ -48,30 +48,26 @@ class ActionsOverview extends Component {
...
-                    <SummaryChartItem
-                        name={ sevNames[key] }
-                        numIssues={ value }
-                        totalIssues={ this.state.total }/>
-                </ConditionalLink>
-            );
-        });
 
-        this.state.category.map((value, key) => donutValues.push([ typeNames[key], value ]));
+        let SummaryChartItems = this.state.severity.map((value, key) =>
```
Comment (AllenBW) [inline src/SmartComponents/Actions/ActionsOverview.js]: last thing, I believe we can const this....
```suggestion
        const SummaryChartItems = this.state.severity.map((value, key) =>
```

Comment (bmclaughlin) [reply src/SmartComponents/Actions/ActionsOverview.js]: Argh, thought I had done that originally...  One sec.

**Pull request merged**